<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>282</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QuickTextEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空内容</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>复制全部</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>54</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>字体大小</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="textsize">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>392</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="_save"/>
    <addaction name="separator"/>
    <addaction name="open_github"/>
    <addaction name="_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>文本复制方式</string>
     </property>
     <property name="icon">
      <iconset resource="appicon.qrc">
       <normaloff>:/deal.ico</normaloff>:/deal.ico</iconset>
     </property>
     <addaction name="copy_all"/>
     <addaction name="separator"/>
     <addaction name="cut_all"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="_clear_all"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="_save">
   <property name="icon">
    <iconset resource="appicon.qrc">
     <normaloff>:/save.ico</normaloff>:/save.ico</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="_exit">
   <property name="icon">
    <iconset resource="appicon.qrc">
     <normaloff>:/exit.ico</normaloff>:/exit.ico</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="copy_all">
   <property name="text">
    <string>复制全部</string>
   </property>
  </action>
  <action name="cut_all">
   <property name="text">
    <string>剪切全部</string>
   </property>
  </action>
  <action name="_clear_all">
   <property name="icon">
    <iconset resource="appicon.qrc">
     <normaloff>:/delete_text.ico</normaloff>:/delete_text.ico</iconset>
   </property>
   <property name="text">
    <string>清空全部 </string>
   </property>
  </action>
  <action name="open_github">
   <property name="icon">
    <iconset resource="appicon.qrc">
     <normaloff>:/Github.ico</normaloff>:/Github.ico</iconset>
   </property>
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="appicon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clear_button</sender>
   <signal>clicked()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textsize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>textEdit</receiver>
   <slot>setFontWeight(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_clear_all</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
